lib/jorb/backends/file.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Jorb.Backends.File do
+  @moduledoc ~S"""
+  File-based backend for light queueing.
+
+  You probably don't want to use this for production load.
```

Comment:
Or with 2+ loads on the same server, unless I'm mistaken.  I get that this is a toy example, but imagine someone using it in tests; they'd likely have to run the tests `async: false`.

Reply:
I don't understand what this comment means

Reply:
Anyone using `Jorb` realistically shouldn't be testing that it does what it says, they would be testing just the contents of their job's `perform` function

Reply:
They wouldn't have to be testing whether Jorb was working correctly; they could choose to use `Jorb.Backends.File` as a backend for tests that involve Jorb in some way.  And the way it's written here, separate invocations of Jorb share the same directory to store queue items, and `ls` is being used to find the top of the queue.  This is going to lead to tests that run concurrently to return strange results.

Reply:
Hm, I don't disagree. Do you have any suggestions?